src/command_modules/azure-cli-batch/setup.py

```diff
@@ -15,7 +15,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "3.0.3+dev"
+VERSION = "3.1.0+dev"
```

Comment:
The version numbers are typically changed at release time.
Since this is automated, I suspect the version no. for this release would be 3.1.1 as we increment the patch version automatically when releasing.